src/language/walker/scopeAwareRuleWalker.ts

```diff
@@ -107,6 +107,6 @@ export abstract class ScopeAwareRuleWalker<T> extends RuleWalker {
     }
 
     protected isScopeBoundary(node: ts.Node): boolean {
-        return isScopeBoundary(node);
+        return isScopeBoundary(node); // tslint:disable-line:deprecation
```

Comment:
can you switch to using the `tsutils` function here?

Reply:
There are some differences between the two that _could_ break some custom rules. Since this is deprecated, I don't think we need to replace all deprecated utils.